modules/ebi-metagenomics/hmmer/hmmsearch/main.nf

```diff
@@ -0,0 +1,67 @@
+
+process HMMER_HMMSEARCH {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
I would say process_medium, since most often we'll give a bunch of CPUs and it takes a while for big datasets